doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1222,6 +1222,33 @@ Zigbee
 
 The issues in this section are related to the :ref:`ug_zigbee` protocol.
 
+.. rst-class:: v2-9-0 v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+NCSIDB-1411: Clearing config data in response to the Leave Network command is not fully performed
+   Configuration data of ZCL Reporting feature is not cleared when processing the "Leave Network" command, which means that compliance with the specification is incomplete.
```

Comment:
```suggestion
NCSIDB-1411: Clearing configuration data is not fully performed when processing the leave network command 
   Configuration data of ZCL Reporting feature is not cleared when processing the leave network command, resulting in incomplete compliance with the specification.
```

Reply:
Proposal to update @annwoj suggestion with Leave Network command by starting the name with capital letter

```suggestion
NCSIDB-1411: Clearing configuration data is not fully performed when processing the Leave Network command 
   Configuration data of ZCL Reporting feature is not cleared when processing the leave network command, resulting in incomplete compliance with the specification.
```

@annwoj  please comment.

Reply:
Should we have he command name highlighted (in double backticks)?